.github/workflows/main-suite.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: yarn install --frozen-lockfile --pure-lockfile
 
       - name: Unit Testing
-        run: yarn test
+        run: yarn test:ci
```

Comment:
here in test matrix we don't need coverage as it is not reported anyway. Make testing a little bit faster.